server/routes/creatingLicences/handlers/checkAnswers.ts

```diff
@@ -18,7 +18,7 @@ export default class CheckAnswersRoutes {
 
   GET = async (req: Request, res: Response): Promise<void> => {
     const { licence, user } = res.locals
-    const backLink = req.session.returnToCase || '/licence/create/caseload'
+    const backLink = req.session?.returnToCase || '/licence/create/caseload'
```

Comment:
There may be a better solution to this, but in the new routing specs (`crdLicenceInHardStop.test.ts`) there is no session to the request. The easiest solution I could find was to handle a null session in a couple of places, as it was blowing up the tests otherwise.

Reply:
👍  
I might try and fix this in a follow up